backend/apps/sandbox/views.py

```
@@ -62,171 +109,60 @@ def get_queryset(self):
-        filename = f"{project.name.replace(' ', '_')}-export.zip"
-        response = HttpResponse(buffer.getvalue(), content_type="application/zip")
-        response["Content-Disposition"] = f'attachment; filename="{filename}"'
-        return response
-
 
 class ProjectFileViewSet(viewsets.ModelViewSet):
+    """
+    ViewSet for ProjectFile model.
+    """
     serializer_class = ProjectFileSerializer
     permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)